src/daemon/daemon_config.cpp

```diff
@@ -121,7 +121,8 @@ std::unique_ptr<const mp::DaemonConfig> mp::DaemonConfigBuilder::build()
         image_hosts.push_back(std::make_unique<mp::UbuntuVMImageHost>(
             std::vector<std::pair<std::string, std::string>>{
                 {mp::release_remote, "https://cloud-images.ubuntu.com/releases/"},
-                {mp::daily_remote, "https://cloud-images.ubuntu.com/daily/"}},
+                {mp::daily_remote, "https://cloud-images.ubuntu.com/daily/"},
+                {mp::appliance_remote, "http://cdimage.ubuntu.com/ubuntu-core/appliances/"}},
```

Comment:
@sil2100 can you please confirm http://cdimage.ubuntu.com/ubuntu-core/appliances/streams/v1/index.json is where these streams will show up? NB: [we're looking for](https://github.com/canonical/multipass/blob/5f90d8e7295464471d853a8421da5d1aaafcc98f/src/simplestreams/simple_streams_manifest.cpp#L125-L141) the `disk1.img` item for qemu and (incoming) for `combined_disk1-img_sha256` under the `lxd.tar.xz` item for LXD.

Reply:
Hey! I think this will be correct, yes. I have my +1 maintenance shift right now but I'll finalize this tomorrow the latest. I'll give my final +1 then.